src/DynamoCoreWpf/UI/GuidedTour/GuidesManager.cs

```diff
@@ -340,17 +340,8 @@ private void CreateGuideSteps(string jsonFile)
         /// <returns></returns>
         private HostControlInfo CreateHostControl(HostControlInfo jsonHostControlInfo)
         {
-            var popupInfo = new HostControlInfo()
-            {
-                PopupPlacement = jsonHostControlInfo.PopupPlacement,
-                HostUIElementString = jsonHostControlInfo.HostUIElementString,
-                HostUIElement = mainRootElement,
-                VerticalPopupOffSet = jsonHostControlInfo.VerticalPopupOffSet,
-                HorizontalPopupOffSet = jsonHostControlInfo.HorizontalPopupOffSet,
-                HtmlPage = jsonHostControlInfo.HtmlPage,
-                WindowName = jsonHostControlInfo.WindowName,
-                DynamicHostWindow = jsonHostControlInfo.DynamicHostWindow
-            };
+            //We use the HostControlInfo copy contrutor
```

Comment:
typo `constructor`

Reply:
@QilongTang  type fixed in the next commit:
https://github.com/DynamoDS/Dynamo/pull/12250/commits/c3995980180bb91f906ebbe6eb891924d0a584ea